helpdesk/teams/models.py

```diff
@@ -58,3 +58,32 @@ class Meta:
 
     def __str__(self) -> str:
         return f"Comment on {self.team.name} at {self.created_at} by {self.author}"
+
+
+class TeamAttendanceEventType(models.TextChoices):
+    ARRIVED = "ARRIVED", "Arrived"
+    LEFT = "LEFT", "Left"
+    DELAYED = "DELAYED", "Delayed"
+    DROPPED_OUT = "DROPPED_OUT", "Dropped Out"
```

Comment:
Thought: worth adding an "unknown" option here?

Might also be worth checking with the comp team whether there are any other events they think might be useful.